modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/exp/ExpressionFactoryImplTest.java

```diff
@@ -176,6 +189,60 @@ public void testRangeConditionWithNullIsIgnored() {
         assertEquals(List.of(new TestRange(new Object[]{1})), list);
     }
 
+    @ParameterizedTest(name = "condition satisfies: [{0}] the index")
```

Comment:
`condition satisfies: [true] the index`
:thinking: Is this an intentional message format?

Reply:
this is not message format and as i know we have no strong suggestions here, if you don`t like it - i can change for more readable - suggest ?